src/content/docs/fr/reference/cli-reference.mdx

```diff
@@ -436,6 +474,10 @@ Active les journaux verbeux, ce qui est utile pour déboguer un problème.
 
 Active les journaux silencieux, ce qui lance le serveur sans aucune sortie dans la console.
 
+### `--open`
+
+Ouvre automatiquement l'application dans le navigateur au démarrage du serveur. Une chaîne d'URL complète (par exemple `--open http://example.com`) ou un chemin d'accès (par exemple `--open /about`) peut être transmis pour spécifier l'URL à ouvrir.
+
 ## Options globales
```

Comment:
```suggestion
## Drapeaux mondiaux
```

It looks like the wrong heading was used.

I want to show Global Flags, but I don't know if the result of the French translation is correct :)

Reply:
Thanks for the suggestion @jsparkdev!
For a literal translation, you would be right: flags should be translated with "drapeaux". But in this context (CLI), I don't think I've ever seen "drapeaux" used in French docs. "options" is more commonly used (and this is what the [French version of Wikipedia](https://fr.wikipedia.org/wiki/Interface_en_ligne_de_commande#Option) or the [PHP doc](https://www.php.net/manual/fr/features.commandline.options.php) uses for example). In other places one can also see "indicateurs" but I'd say it is less common than "option".

So I think the current version ("Options globales") is the correct translation for "Global Flags"!

Reply:
Oh I see, thank you for your response!